src/Tracker.js

```diff
@@ -57,10 +57,10 @@ function _debugFunc() {
   return typeof Meteor !== 'undefined'
     ? Meteor._debug
     : typeof console !== 'undefined' && console.error
-    ? function () {
-        console.error.apply(console, arguments);
-      }
-    : function () {};
+      ? function () {
+          console.error.apply(console, arguments);
+        }
+      : function () {};
 }
```

Comment:
I think the linter has some weird understanding of indentation on nested ternary operators. Maybe this one might work:

```js
if (typeof Meteor !== 'undefined') {
  return Meteor._debug;
}

return typeof console !== 'undefined' && console.error
  ? function () { console.error.apply(console, arguments); }
  : function () {};
```